community/cypher/cypher-compiler-2.2/src/main/scala/org/neo4j/cypher/internal/compiler/v2_2/spi/GraphStatistics.scala

```diff
@@ -28,6 +28,7 @@ object GraphStatistics {
   val DEFAULT_EQUALITY_SELECTIVITY       = Selectivity(0.1)
   val DEFAULT_NUMBER_OF_ID_LOOKUPS       = Cardinality(25)
   val DEFAULT_NUMBER_OF_INDEX_LOOKUPS    = Cardinality(25)
+  val DEFAULT_LIMIT_CARDINALITY          = Cardinality(75)
```

Comment:
No value is a good value, but leaving it at incoming cardinality is even worse, so this is better than that, at least. A better idea is to not auto-parameterize LIMIT literals, since they probably cover a limited range of values only. If that is done, this value would only be here to catch the rare cases of manual parameterization of LIMIT.